packages/internal-test-helpers/lib/ember-dev/run-loop.ts

```diff
@@ -1,14 +1,14 @@
 import { end, _cancelTimers, _getCurrentRunLoop, _hasScheduledTimers } from '@ember/runloop';
 
 export function setupRunLoopCheck(hooks: NestedHooks) {
-  hooks.afterEach(function (assert) {
+  hooks.afterEach(function (assert: QUnit['assert']) {
     if (_getCurrentRunLoop() || _hasScheduledTimers()) {
       let done = assert.async();
       // use a setTimeout to allow the current run loop to flush via autorun
       setTimeout(() => {
         // increment expected assertion count for the assertions just below
-        if (assert['test'].expected !== null) {
-          assert['test'].expected += 2;
+        if ((assert as any)['test'].expected !== null) {
+          (assert as any)['test'].expected += 2;
```

Comment:
Weird, is `['test']` missing from the types for QUnit proper?

Reply:
Correct.